docs/releases/ez_platform_v3.0_deprecations.md

```diff
@@ -159,6 +164,23 @@ Additionally, the listener for `pbPreviewReloaded` has been removed.
 |`fbFormBuilderLoaded`|`ez-form-builder-loaded`|
 |`fbFormBuilderUnloaded`|`ez-form-builder-unloaded`
 
+#### Code cleanup
+
+Selected deprecated functions have been dropped.
```

Comment:
> deprecated functions

In the table below we have functions but also React components and components' properties.